packages/components/pie-icon-button/test/accessibility/pie-icon-button.spec.ts

```diff
@@ -5,26 +5,45 @@ import type {
 import {
     getAllPropCombinations, splitCombinationsByPropertyValue,
 } from '@justeattakeaway/pie-webc-testing/src/helpers/get-all-prop-combos.ts';
+import { IconClose } from '@justeattakeaway/pie-icons-webc';
 import { PieIconButton } from '../../src/index.ts';
 import { variants } from '../../src/defs.ts';
 
 const props: PropObject = {
     variant: variants,
     disabled: [true, false],
+    aria: {
+        labelledby: 'foo',
+        describedby: 'bar',
+        controls: 'baz',
+        expanded: false,
+        label: 'qux',
+    },
 };
 
 const componentPropsMatrix: WebComponentPropValues[] = getAllPropCombinations(props);
 const componentPropsMatrixByVariant: Record<string, WebComponentPropValues[]> = splitCombinationsByPropertyValue(componentPropsMatrix, 'variant');
 const componentVariants: string[] = Object.keys(componentPropsMatrixByVariant);
 
+test.beforeEach(async ({ mount }, testInfo) => {
+    testInfo.setTimeout(testInfo.timeout + 40000);
```

Comment:
I see there other occurrences of this increased setTimeout, do we have plans to tidy up with some shared constant for the cases where the test might take longer?

Reply:
I think this would be good to discuss offline and we can include @JoshuaNg2332 as I am less familiar with why we are doing this

Reply:
Created a discussion in our "messaging app".